Polynomials.Core/Polynomial.cs

```diff
@@ -0,0 +1,237 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Polynomials.Core
+{
+    public class Polynomial:IEquatable<Polynomial>
+    {
+        private readonly float[] _coefficients;
+
+        public float this[int i]
+        {
+            get => _coefficients[i];
+            set => _coefficients[i] = value;
+        }
+
+        public Polynomial(float[] coefficients)
+        {
+            if (coefficients.Length>100)
```

Comment:
Please add spaces before and after the > sign

Reply:
Done.